vm/page.c

```diff
@@ -103,11 +106,12 @@ void _page_free(page_t *p)
 	page_t *lh = p, *rh = p;
 
 #if 1
```

Comment:
Not technically a dead code, but is this necessary?

Reply:
imho may be removed, it's not like anyone remembers why is this way it is